### ISSUE TITLE
Firefox error: "Project key not found on window"

### ISSUE THREAD

**Issue by michaelfester** [motifland/markprompt#213]
**Describe the bug**

<img width="549" alt="CleanShot 2023-10-03 at 22 31 55@2x" src="https://github.com/motifland/markprompt/assets/504893/0ecca7dd-07b0-477f-8f02-423768b6bbce">

Link: https://ddk-dash-docs-test-markprompt-f6556780ed65.herokuapp.com/

**To Reproduce**

1. Add Markprompt with the script tag in an HTML page
2. Open the page in Firefox
3. See console error, and bot icon not showing up

**Comment by nickrttn** [comment]
This gist outlines nicely why this is happening: https://gist.github.com/jakub-g/385ee6b41085303a53ad92c7c8afd7a6

In short, `async` scripts with `type=module` are executed immediately, while regular `type=module` scripts are deferred until the whole DOM has been parsed.

TL;DR is, in the link shared the `async` attribute should be removed from the script tag that loads Markprompt, and all should be well. We do not show the `async` attribute in the README, but it might still be on our `/docs` page?
